packages/dm-core-plugins/src/header/HeaderPlugin.tsx

```diff
@@ -28,7 +27,10 @@ const Icons = styled.div`
   }
 `
 
-const ClickableIcon = styled.div`
+const ClickableIcon = styled.button`
```

Comment:
I think this component if pre-EDS. The styling is wrong and it lacks a aria-label. We should instead use something like the TooltipButton I made.